### DISCUSSION TITLE
Key for Project retail 2021 not working

### DISCUSSION THREAD

**Discussion by Christopha1976** [YerongAI/Office-Tool#821] [Q&A]
Hi. Do you have updated key fo Project 2021 retail? Current one is not working anymore.

By the ay, great work. Thanks

**Comment by YerongAI** [comment]
I have tried standard and professional version, both works fine.

![image](https://github.com/YerongAI/Office-Tool/assets/44803668/d798feec-2448-4d6b-b518-36111d0c2929)

The key is grace key, you need to buy a genuine key from MS to activate your Project 2021 Retail.
